tests/testcase_generators/testcase_generator.py

```diff
@@ -384,10 +385,14 @@ def generate_test_case(self):
             )
 
         parameters = input_["parameters"]
+
         try:
             output = self.compute_expected_output(input_data, parameters)
+
         except Exception as err:
-            raise Exception(f"{err}, datasets: {input_['inputdata']['datasets']}")
+            raise Exception(
+                f"{err}, input_data: {input_data}, parameters: {parameters}, datasets: {input_['inputdata']['datasets']}, traceback: {format_exc()}"
+            )
```

Comment:
I don't think I should be cherry picking so much, we will end up with double commits, also I have already generated the expected file. For the purpose of this specific branch, I think this can be left as is, and this can be merged with master when the branch that contains it is merged.